### ISSUE TITLE
Posts List: list view and add buttons are read incorrectly with VoiceOver

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12872]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to My Site > Blog Posts.
1. Swipe right to reach the buttons in the header for toggling compact and expanded view modes and for adding a new post.

**Result:** the compact list view button is read as "Button. Possibly menu." and should be something like "Switch to compact list view. Button." the expanded list view button is read as "Button." and should be something like "Switch to expanded list view. Button." and the add buton is read as "Button. Possibly add." and should be something like "Add new post. Button." ([5s](https://cloudup.com/cD6_Yuu34pZ))

<img src="https://user-images.githubusercontent.com/1119271/68252593-28e0f000-ffe3-11e9-8283-c903243b5e9e.png" width="270" alt="posts-list-add-button">
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

**Comment by vphong** [comment]
Hey, I was looking into this and added some lines just to see if it would work. #13038 I tested using the Accessibility Inspector, so I'm not sure if it will read the same on a physical device. Also, I'm not sure if the way I wrote my solution is acceptable considering there's a lot of `NSLocalizedString`s around the code. I tried something like:
```swift
addButton.accessibilityLabel = NSLocalizedString("Add Button", comment: "Add new post")
```
at first to follow the other similar cases, but the Accessibility Inspector would only read "Add Button".

**Comment by shiki** [comment]
Hi @vphong! I responded to your question about `NSLocalizedString` in the PR (#13038). In general, as long as you use `NSLocalizedString()`, that should be enough. 🙂 The translation will be handled later. More information about that [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/CONTRIBUTING.md#translating).
